docs/CHANGELOG.asciidoc

```diff
@@ -97,6 +98,70 @@ Review important information about the {kib} 8.x releases.
 
 include::upgrade-notes.asciidoc[]
 
+[[release-notes-8.18.3]]
```

Comment:
One more change, plus the suggestion I posted in Slack about updating [this](https://github.com/elastic/kibana/pull/224482/files#diff-6eabb97e98640429cb68b61823f99438d287d604d72ecf0e93ed2e9ecbec7479R174-R196) known issue entry show that it was fixed in 8.18.3.

```suggestion
[float]
[[known-issues-8.18.3]]
=== Known issues
include::CHANGELOG.asciidoc[tag=known-issue-1508]


[[release-notes-8.18.3]]
```

Reply:
Thanks for the note. This is inserting the content outside of the 8.18.3 section so I've done it in [9a7649e](https://github.com/elastic/kibana/pull/224482/commits/9a7649eb893eec4137ad3b074614d178a5c52101).